crates/typst/src/visualize/line.rs

```diff
@@ -89,6 +89,7 @@ impl Layout for LineElem {
         let mut frame = Frame::soft(target);
         let shape = Geometry::Line(delta.to_point()).stroked(stroke);
         frame.push(start.to_point(), FrameItem::Shape(shape, self.span()));
+        frame.meta(styles, false);
```

Comment:
I dont really understant if I should pass true/false here. Please check

Reply:
false is good